tests/drivers/disk/disk_performance/boards/rcar_h3ulcb_ca57.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2023 EPAM Systems
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SDHC=y
+CONFIG_MMC_RCA=1
```

Comment:
Where is Kconfig option MMC_RCA introduced?

Reply:
okay, with Kconfig options introduced here MMC_RCAR and RCAR_MMC_*, this MMC_RCA is very confusing.
Anyway, @danieldegrasse if a driver is designed to be the same for SD/MMC and eMMC controllers, then it looks like option MMC_RCA should not be Kconfig option.

Reply:
> okay, with Kconfig options introduced here MMC_RCAR and RCAR_MMC_*, this MMC_RCA is very confusing.
Anyway, @danieldegrasse if a driver is designed to be the same for SD/MMC and eMMC controllers, then it looks like option MMC_RCA should not be Kconfig option.

It is accurate that MMC_RCA should probably not be a Kconfig option. However, there is a bit more that would be required to support multiple cards on an MMC/SD bus. Once a card is in data transfer mode (after SET_RCA/CMD3 is sent), we could switch between active cards with CMD7 (GO_TRANSFER_STATE). This would require some additional changes to the subsystem, but I can implement it if I can find hardware that will enable testing multiple SD devices on one bus.

For the time being, I don't think this issue is in scope for this PR